packages/corelib/src/pubsub.ts

```diff
@@ -0,0 +1,209 @@
+import { RundownBaselineAdLibItem } from './dataModel/RundownBaselineAdLibPiece'
+import { DBPartInstance } from './dataModel/PartInstance'
+import { DBRundown } from './dataModel/Rundown'
+import { DBRundownPlaylist } from './dataModel/RundownPlaylist'
+import { DBSegment } from './dataModel/Segment'
+import { DBShowStyleBase } from './dataModel/ShowStyleBase'
+import { DBShowStyleVariant } from './dataModel/ShowStyleVariant'
+import { DBStudio } from './dataModel/Studio'
+import { IngestDataCacheObj } from './dataModel/IngestDataCache'
+import { DBTimelineDatastoreEntry } from '@sofie-automation/shared-lib/dist/core/model/TimelineDatastore'
+import { Blueprint } from './dataModel/Blueprint'
+import { BucketAdLibAction } from './dataModel/BucketAdLibAction'
+import { BucketAdLib } from './dataModel/BucketAdLibPiece'
+import { ExpectedMediaItem } from './dataModel/ExpectedMediaItem'
+import { ExpectedPackageWorkStatus } from './dataModel/ExpectedPackageWorkStatuses'
+import { ExpectedPackageDB, ExpectedPackageDBBase } from './dataModel/ExpectedPackages'
+import { ExternalMessageQueueObj } from './dataModel/ExternalMessageQueue'
+import { PackageContainerStatusDB } from './dataModel/PackageContainerStatus'
+import { PeripheralDevice } from './dataModel/PeripheralDevice'
+import { Piece } from './dataModel/Piece'
+import { PieceInstance } from './dataModel/PieceInstance'
+import { TimelineComplete } from './dataModel/Timeline'
+import {
+	RundownId,
+	RundownPlaylistId,
+	ShowStyleBaseId,
+	StudioId,
+} from '@sofie-automation/shared-lib/dist/core/model/Ids'
+import { RundownPlaylistActivationId } from './dataModel/Ids'
+
+export enum CorelibPubSub {
+	rundownPlaylists = 'rundownPlaylists',
+	rundowns = 'rundowns',
+	ingestDataCache = 'ingestDataCache',
+
+	rundownBaselineAdLibPieces = 'rundownBaselineAdLibPieces',
+	rundownBaselineAdLibActions = 'rundownBaselineAdLibActions',
+	adLibActions = 'adLibActions',
+	adLibPieces = 'adLibPieces',
+
+	segments = 'segments',
+	parts = 'parts',
+	partInstances = 'partInstances',
+	partInstancesSimple = 'partInstancesSimple',
+	partInstancesForSegmentPlayout = 'partInstancesForSegmentPlayout',
+	pieces = 'pieces',
+	pieceInstances = 'pieceInstances',
+	pieceInstancesSimple = 'pieceInstancesSimple',
+
+	timeline = 'timeline',
+	timelineDatastore = 'timelineDatastore',
+
+	expectedMediaItems = 'expectedMediaItems',
+
+	expectedPackages = 'expectedPackages',
+	expectedPackageWorkStatuses = 'expectedPackageWorkStatuses',
+	packageContainerStatuses = 'packageContainerStatuses',
+
+	bucketAdLibPieces = 'bucketAdLibPieces',
+	bucketAdLibActions = 'bucketAdLibActions',
+
+	externalMessageQueue = 'externalMessageQueue',
+
+	blueprints = 'blueprints',
+	showStyleBases = 'showStyleBases',
+	showStyleVariants = 'showStyleVariants',
+	studios = 'studios',
+
+	peripheralDevices = 'peripheralDevices',
+	peripheralDevicesAndSubDevices = ' peripheralDevicesAndSubDevices',
```

Comment:
typo? (there's a space in there)

Reply:
That appears to be an existing typo, but it may as well be fixed now